code/modules/mob/living/carbon/human/login.dm

```diff
@@ -0,0 +1,23 @@
+/mob/living/carbon/human/Login()
+	. = ..()
+	if(!LAZYLEN(afk_thefts))
+		return
+
+	var/list/print_msg = list()
+	print_msg += "<span class='info'>*---------*</span>"
+	print_msg += "<span class='userdanger'>As you snap back to consciousness, you recall people messing with your stuff...</span>"
+
+	for(var/i in afk_thefts)
+		var/list/iter_theft = i
+		if(!islist(iter_theft) || LAZYLEN(iter_theft) != 2)
```

Comment:
I don't think you should have these checks, these seem like cases where you should fail fast.

Reply:
Just so I can be as clear as possible, if it shouldn't be possible for a content in afk_thefts to *not* be a 2 entry list, then we should not pretend in code like it can be. If something comes up that *is*, then this should either be a check that `CRASH`es, or simply let it runtime, but it shouldn't be silenced.

Reply:
Alright cool, I wasn't sure and did that just to be safe, but doing CRASH sounds better